scenario/CrossChainGovernanceScenario.ts

```diff
@@ -262,6 +261,114 @@ scenario(
   }
 );
 
+scenario(
+  'test Succinct cross-chain governance proposals',
```

Comment:
Just to be consistent with the other tests, I would rename as: `upgrade Succinct bridge governance contracts and ensure they work properly`